styles.css

```diff
@@ -92,6 +92,10 @@ body {
   width: 100%;
 }
 
+.shuffle-button {
+  margin-left: 54px;
```

Comment:
```suggestion

```
This looks good when the site's in thin responsive mode, but when open big on a browser for me it wraps the button underneath the others. Really all these buttons should be fixed with a flexbox. Using no margin just sticks it after `Change name` which is OK imo

Reply:
Works for me!

Reply:
hm, worked for me for like a second and then I reloaded and it started wrapping

Reply:
You ok with this suggested change?